proto/common.proto

```diff
@@ -130,13 +130,13 @@ message Location {
 }
 
 message TransitEgressInfo {
-  string onestop_id = 1;        // Unique transitland ID
+  string onestop_id = 1;        // Unique ID
```

Comment:
imho we should rename this when we do the work of coming up with our predictable id